stylesheets/components/badges.css

```diff
@@ -0,0 +1,91 @@
+.badge-wrapper {
+  background-color: var(--baltic-sea-color);
+  border: 1px solid var(--charcoal-grey);
+  border-radius: 6px;
+  display: flex;
+  min-width: 16rem;
+  justify-content: space-around;
+  padding: 1rem 1.5rem;
+}
+.badge {
+  position: relative;
+}
+.badge-round{
+    position: absolute;
+    z-index: 1;
+    border-radius: 50%;
+}
+/* icons with badge */
+.badge-icon {
+  color: var(--blue-semi-dark);
+  font-size: 2.5rem !important;
```

Comment:
You can try to do this without `!important`.
According to me `!important` should be the last option in the list to use.

Reply:
Initially, I tried doing it without `!important` but it wasn't working, so I had to use `!important`